stem/IO/kratos_loads_io.py

```diff
@@ -0,0 +1,101 @@
+from typing import Dict, List, Any
+from copy import deepcopy
+
+import numpy as np
+
+
+from stem.load import Load, PointLoad, MovingLoad
+
+
+class KratosLoadsIO:
+
+    def __init__(self, domain: str):
+        self.domain = domain
+        pass
+
+    def __create_point_load_dict(self, load: Load) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the point load parameters
+
+        Args:
+            load (Load): point load object
+
+        Returns:
+            Dict[str, Any]: dictionary containing the load parameters
+        """
+
+        # initialize load dictionary
+        load_dict: Dict[str, Any] = {
+            "python_module": "apply_vector_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name": "ApplyVectorConstraintTableProcess",
+            "Parameters": deepcopy(load.load_parameters.__dict__),
+        }
+
+        load_dict["Parameters"]["model_part_name"] = f"{self.domain}.{load.name}"
+        load_dict["Parameters"]["variable_name"] = "POINT_LOAD"
+        load_dict["Parameters"]["table"] = [0, 0, 0]
+
+        return load_dict
+
+    def __create_moving_load_dict(self, load: Load) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the moving load parameters
+
+        Args:
+            load (Load): moving load object
+
+        Returns:
+            Dict[str, Any]: dictionary containing the load parameters
+        """
+
+        # initialize load dictionary
+        load_dict: Dict[str, Any] = {
+            "python_module": "set_moving_load_process",
+            "kratos_module": "StructuralMechanicsApplication",
+            "process_name": "SetMovingLoadProcess",
+            "Parameters": deepcopy(load.load_parameters.__dict__),
+        }
+
+        load_dict["Parameters"]["model_part_name"] = f"{self.domain}.{load.name}"
+        load_dict["Parameters"]["variable_name"] = "POINT_LOAD"
+
+        return load_dict
+
+    def __create_load_dict(self, load: Load) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the load parameters
+
+        Args:
+            load (Load): load object
+
+        Returns:
+            Dict[str, Any]: dictionary containing the load parameters
+        """
+
+        # add load parameters to dictionary based on load type.
+        if isinstance(load.load_parameters, PointLoad):
+            return self.__create_point_load_dict(load=load)
+        elif isinstance(load.load_parameters, MovingLoad):
+            return self.__create_moving_load_dict(load=load)
+        else:
+            raise NotImplementedError
+
+    def create_loads_process_dictionary(self, loads: List[Load]) -> Dict[str, Any]:
```

Comment:
sometimes you call a function ..._dict sometimes ..._dictionary. please be uniform